esmvaltool/diag_scripts/kcs/global_matching.py

```diff
@@ -0,0 +1,187 @@
+"""Align the target model with the CMIP ensemble."""
+import logging
+from pathlib import Path
+from itertools import product
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import xarray as xr
+
+from esmvaltool.diag_scripts.shared import (get_diagnostic_filename,
+                                            get_plot_filename, run_diagnostic,
+                                            select_metadata, ProvenanceLogger)
+
+logger = logging.getLogger(Path(__file__).name)
+
+
+def create_provenance_record(ancestor_files):
+    """Create a provenance record."""
+    record = {
+        'caption': "Resampling of local climate model.",
+        'domains': ['global'],
```

Comment:
Would `'reg'` not be more applicable to a local climate model?

Reply:
This is global. Changed the caption should change instead, (this was probably an old version).